Tests/Twig/ExtensionTest.php

```diff
@@ -73,7 +79,7 @@ public function testTrackSiteSearchDisablesPageTracking()
 
     public function testIsTwigExtension()
     {
-        $extension = new Extension(false, 1, '', false);
+        $extension = new Extension(false, 1, '', false, true);
         $this->assertInstanceOf('\Twig_ExtensionInterface', $extension);
```

Comment:
Could you change this string to be `AbstractExtension::class` and import that as `Twig\Extension\AbstractExtension`? Hopefully that would make the tests pass.